### PR TITLE
[Merged by Bors] - feat(Condensed): the functor from `TopCat` to `LightCondSet`

### PR DESCRIPTION
**Pull request by dagurtomas**
- [x] depends on: #13503 

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Condensed/Light/TopComparison.lean [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+/-!
+
+# The functor from topological spaces to light condensed sets
+
+We define the functor `topCatToLightCondensed : TopCat.{u} ⥤ CondensedSet.{u}`.
+
+## Projects
+
+* Prove that `topCatToLightCondensed` is faithful.
```
Comment (semorrison) [inline Mathlib/Condensed/Light/TopComparison.lean:17]: `TopCat.toLightCondensed`

and above and below

File: Mathlib/Condensed/Light/TopComparison.lean [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+universe w w' v u
+
+open CategoryTheory Opposite Limits regularTopology ContinuousMap
+
+/--
+Associate to a `u`-small topological space the corresponding light condensed set, given by
+`yonedaPresheaf`.
+-/
+-- @[simps!]
+noncomputable def TopCat.toLightCondensed (X : TopCat.{u}) : LightCondSet.{u} :=
```
Comment (semorrison) [inline Mathlib/Condensed/Light/TopComparison.lean:32]: `toLightCondSet`?

similarly below

File: Mathlib/Condensed/Light/TopComparison.lean [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+Authors: Dagur Asgeirsson
+-/
+import Mathlib.Condensed.Light.Explicit
+import Mathlib.Condensed.TopComparison
+
+/-!
+
+# The functor from topological spaces to light condensed sets
+
+We define the functor `topCatToLightCondensed : TopCat.{u} ⥤ CondensedSet.{u}`.
```
Comment (semorrison) [inline Mathlib/Condensed/Light/TopComparison.lean:13]: Target category name is incorrect.

**Comment by semorrison** [comment]
bors d+

**Comment by dagurtomas** [comment]
Thanks! 

bors merge

**Pull request closed without merging**